app/views/hosts/_progress.html.erb

```diff
@@ -1,8 +1,8 @@
-<div id="host-progress" class="alert alert-block alert-success base in fade hide alert-dismissable">
+<div id="host-progress" class="alert alert-block alert-info base in fade hide alert-dismissable">
   <%= alert_close %>
-  <h3><%= action_name.capitalize %> <%= _('in Progress') %></h3>
-  <p id="tasks_progress"><%= action_name %> in progress.</p>
-  <div class="progress progress-striped active">
+  <h3><span class='pficon pficon-info'></span> <%= action_name.capitalize %> <%= _('in Progress') %></h3>
```

Comment:
Added helper for it, but there are tons of places we can use icon helper method.
opened an issue: http://projects.theforeman.org/issues/12972